core/src/main/scala/org/apache/spark/metrics/sink/PrometheusServlet.scala

```diff
@@ -23,23 +23,19 @@ import javax.servlet.http.HttpServletRequest
 import com.codahale.metrics.MetricRegistry
 import org.eclipse.jetty.servlet.ServletContextHandler
 
-import org.apache.spark.{SecurityManager, SparkConf}
-import org.apache.spark.annotation.Experimental
+import org.apache.spark.SparkConf
 import org.apache.spark.ui.JettyUtils._
 
 /**
- * :: Experimental ::
```

Comment:
@dongjoon-hyun, would you mind double checking this? I am reading https://github.com/apache/spark/commit/b80309bdb4d26556bd3da6a61cac464cdbdd1fe1#. Maybe we could just remove `Experimental` here. Seems like the endpoints are documented as experimental properly as intended, and this `PrometheusServlet` class seems not an API.

Reply:
+1 for removal, but not in this PR because this PR's title is `[SPARK-34520][CORE] Remove unused SecurityManager references`.

Reply:
It is actually related. If this is marked as an API, I should deprecate the security manager first..

Reply:
Sure 👍